client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { translate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Card } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import CardHeading from 'components/card-heading';
+import { getTaskList } from 'lib/checklist';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
+import { getSiteOption } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import NavItem from './nav-item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const getTaskCopy = ( task ) => {
+	switch ( task ) {
+		case 'domain_verified':
+			return {
+				title: translate( 'Verify the email address for your domains' ),
+			};
+		case 'email_verified':
+			return {
+				title: translate( 'Confirm your email address' ),
+			};
+		case 'blogname_set':
+			return {
+				title: translate( 'Name your site' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **47** times:
`translate( 'Name Your Site' )` *ES Score: 7*